app/src/main/kotlin/com/ttvnp/ttj_asset_android_client/presentation/ui/fragment/SettingsProfileEditFragment.kt

```diff
@@ -271,6 +275,91 @@ class SettingsProfileEditFragment : BaseMainFragment(), SettingsProfileEditPrese
         return identified
     }
 
+    private fun checkUnderReviewing(status: Int) {
+        if (status == IdentificationStatus.Applied.rawValue) {
+            var view:View? = null
+            context?.let {
+                val textColor = ContextCompat.getColor(it, R.color.md_grey_400)
+                textProfileFirstName.setTextColor(textColor)
+                textProfileFirstName.onFocusChangeListener = onFocusChangeListener { v ->
+                    clearUnderReviewMessage(view)
+                    view = v
+                }
+                textProfileMiddleName.setTextColor(textColor)
+                textProfileMiddleName.onFocusChangeListener = onFocusChangeListener { v ->
+                    clearUnderReviewMessage(view)
+                    view = v
+                }
+                textProfileLastName.setTextColor(textColor)
+                textProfileLastName.onFocusChangeListener = onFocusChangeListener { v ->
+                    clearUnderReviewMessage(view)
+                    view = v
+                }
+                textProfileAddress.setTextColor(textColor)
+                textProfileAddress.onFocusChangeListener = onFocusChangeListener { v ->
+                    clearUnderReviewMessage(view)
+                    view = v
+                }
+                textProfileCellPhoneNumberNationalCode.setTextColor(textColor)
+                textProfileCellPhoneNumberNationalCode.onFocusChangeListener = onFocusChangeListener {
+                    clearUnderReviewMessage(view)
+                    textProfileCellPhoneNumberNationalCode.error = null
+                    textProfileCellPhoneNumber.error = getString(R.string.message_under_review)
+                    textProfileCellPhoneNumber.requestFocus()
+                    view = textProfileCellPhoneNumber
+                }
+                textProfileCellPhoneNumber.setTextColor(textColor)
+                textProfileCellPhoneNumber.onFocusChangeListener = onFocusChangeListener {
+                    clearUnderReviewMessage(view)
+                    textProfileCellPhoneNumber.error = getString(R.string.message_under_review)
+                    view = it
+                }
+                radioGroupGender.setOnClickListener {
+                    clearUnderReviewMessage(view)
+                    Toast.makeText(context, getString(R.string.message_under_review), Toast.LENGTH_SHORT).show()
+                }
+                radioFemale.isEnabled = false
+                radioMale.isEnabled = false
+                textDOB.setTextColor(textColor)
+                textDOB.setOnClickListener {
+                    clearUnderReviewMessage(view)
+                    textDOB.requestFocus()
+                    Toast.makeText(context, getString(R.string.message_under_review), Toast.LENGTH_SHORT).show()
+                }
+            }
+        }
+    }
+
+    private fun setEnableView(b: Boolean) {
```

Comment:
@Thisak I don't like this attribute name :(

Reply:
okay